components/InfoPane/InfoPane.tsx

```diff
@@ -37,6 +37,31 @@ export default function InfoPane(props: HTMLProps<HTMLDivElement>) {
           <FilePicker label="Choose file" /> from your computer
         </li>
       </ul>
+
+      <hr />
+      <footer>
+        <p>
+          npmgraph
+          {' | '}
```

Comment:
No real reason for having this here anymore, right?
```suggestion
```

Reply:
I was looking for a way to make sense of the footer, but we can drop it for now. 

The _documentation_ above the footer is now the uglier part.